src/Microsoft.Health.Fhir.TemplateManagement.FunctionalTests/OrasUtility.cs

```diff
@@ -0,0 +1,38 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Diagnostics;
+using System.IO;
+
+namespace Microsoft.Health.Fhir.TemplateManagement.FunctionalTests
+{
+    public static class OrasUtility
+    {
+        public static string OrasExecution(string command, string orasFileName)
+        {
+            Process process = new Process
+            {
+                StartInfo = new ProcessStartInfo(orasFileName),
+            };
+
+            process.StartInfo.Arguments = command;
+            process.StartInfo.RedirectStandardError = true;
+            process.EnableRaisingEvents = true;
+            process.Start();
+
+            StreamReader errStreamReader = process.StandardError;
+            process.WaitForExit(30000);
```

Comment:
The constant value `30000` can be extracted out as a static readonly class field for unified management.

Reply:
Constants is an internal class could not be used here. Maybe hard code here is fine in test project?

Reply:
I see. I believe the hard coding for tests is ok here.